app/models/env_var.rb

```diff
@@ -0,0 +1,3 @@
+class EnvVar < ApplicationRecord
```

Comment:
Id recommend using the full name. I know it can be verbose but you shouldn't shorten class names.